db/migrate/20160916063059_parse_double_encoded_json.rb

```diff
@@ -0,0 +1,18 @@
+class ParseDoubleEncodedJson < ActiveRecord::Migration
+  def change
+    ids = CustomField.where(data_type: 'breakeven_data').ids.map(&:to_s)
+    puts "CustomField ids: #{ids.inspect}"
+    ids.each do |id|
+      lrss = LoanResponseSet.select { |i| i.custom_data.keys.include? id }
```

Comment:
I prefer not to have AR classes in migrations wherever possible as they tend to lead to migration failures. But at the very least can you please add

```
class CustomField < ActiveRecord::Base; end
class LoanResponseSet < ActiveRecord::Base; end
```

to the top of the file?